net-print/kyocera-1x2x-mfp-driver/kyocera-1x2x-mfp-driver-1.1203-r1.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Printer descriptions (PPDs) and filters for Kyocera 1x2x MFP"
+HOMEPAGE="http://www.kyoceramita.it/"
+SRC_URI="LinuxDrv_${PV}_FS-1x2xMFP.zip"
+
+LICENSE="kyocera-mita-ppds"
+SLOT="0"
+
+# TODO: Someone please test the ebuild on ~x86.
+KEYWORDS="-* ~amd64 ~x86"
+
+IUSE_LINGUAS=(ar cs de el en es fr he hu it ko pl pt ro ru th tr vi zh_CN zh_TW)
+IUSE="+rastertokpsl-fix ${IUSE_LINGUAS[@]/#/linguas_}"
+REQUIRED_USE="|| ( ${IUSE_LINGUAS[@]/#/linguas_} )"
+
+RDEPEND="net-print/cups"
+DEPEND="app-arch/unzip"
+RESTRICT="fetch"
+
+S="${WORKDIR}/Linux"
+
+function archSubdirectory {
+	if use x86 && ! use amd64
+	then echo "32bit"
+	else echo "64bit" ; fi
+}
+
+function languageArchiveName {
+	# Note the capitalisation inconsistency. Don't "fix" that.
+	case "$1" in
+		ar)    echo arabic      ;;
+		cs)    echo czech       ;;
+		de)    echo German      ;;
+		el)    echo greek       ;;
+		en)    echo English     ;;
+		es)    echo Spanish     ;;
+		fr)    echo French      ;;
+		he)    echo hebrew      ;;
+		hu)    echo hungarian   ;;
+		it)    echo Italian     ;;
+		ko)    echo Korean      ;;
+		pl)    echo polish      ;;
+		pt)    echo Portuguese  ;;
+		ro)    echo romanian    ;;
+		ru)    echo russian     ;;
+		th)    echo thai        ;;
+		tr)    echo turkish     ;;
+		vi)    echo vietnamese  ;;
+		zh_CN) echo simplified  ;;
+		zh_TW) echo traditional ;;
+	esac
+}
+
+pkg_nofetch() {
+	einfo "Please, navigate your browser to the following URL and manually"
+	einfo "download the file named '$A', then put it into '$DISTDIR'."
+	einfo "http://www.kyoceradocumentsolutions.eu/index/service/dlc.false.driver.FS1025MFP._.EN.html"
+	einfo ""
+	einfo "Consider keeping a local copy of the file since there're chances"
+	einfo "the company is going to eventually stop hosting it for whatever"
+	einfo "reason."
+}
+
+src_prepare() {
+	eapply_user
+
+	function patchFilterPath {
+		local orig="/usr/lib/cups/filter/rastertokpsl"
+		local repl="/usr/libexec/cups/filter/rastertokpsl"
+		use rastertokpsl-fix &&
+			repl="${repl}-fix.sh"
+
+		sed -i "s|$orig|$repl|g" "$@"
+	}
+
+	local g="$S/$(archSubdirectory)/Global"
+	local lingua
+	for lingua in "${IUSE_LINGUAS[@]}"; do
+		use linguas_$lingua || continue
+
+		local language="$(languageArchiveName "$lingua")"
+
+		unpack "$g/$language.tar.gz"
```

Comment:
Unfortunately, the file provided by Kyocera is a ZIP archive containing a number of tarballs (one for each language). Does unpacking nested tarballs count as 'unpack' or 'prepare'? We don't potentially need to unpack them all. Is it okay for `src_unpack` to depend on use flags?